packages/app-mobile/root.tsx

```diff
@@ -215,7 +215,9 @@ const generalMiddleware = (store: any) => (next: any) => async (action: any) =>
 	}
 
 	if (action.type === 'NAV_GO' && action.routeName === 'Notes') {
-		Setting.setValue('activeFolderId', newState.selectedFolderId);
+		if (newState.selectedFolderId) {
+			Setting.setValue('activeFolderId', newState.selectedFolderId);
+		}
```

Comment:
There should be a comment describing this change. Currently, it isn't clear that `selectedFolderId` should only change if present in `newState`.

Alternatively, something similar to the following could be clearer:
```suggestion
		if ('selectedFolderId' in newState) {
			Setting.setValue('activeFolderId', newState.selectedFolderId);
		}
```

Reply:
Changed.